.github/workflows/test.yaml

```diff
@@ -0,0 +1,15 @@
+name: Test
+
+on:
+  push:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '>=1.18.0'
```

Comment:
Maybe this action too so we can control the terraform version. Otherwise we'll be stuck with the default version github installs.

https://github.com/hashicorp/setup-terraform

Reply:
pinned to latest minor